groups/sig-network/groups.yaml

```diff
@@ -152,6 +152,19 @@ groups:
       - longwuyuan@gmail.com
       - zhangjintao9020@gmail.com
 
+  - email-id: k8s-infra-staging-ingate@kubernetes.io
+    name: k8s-infra-staging-ingate
+    description: |-
+      ACL for pushing ingate artifacts
+    settings:
+      ReconcileMembers: "true"
+    members:
+      - cpanato@gmail.com
+      - thockin@google.com
+      - strong.james.e@gmail.com
+      - longwuyuan@gmail.com
+      - zhangjintao9020@gmail.com
```

Comment:
@strongjz can you check the emails? add/remove who?

Reply:
all good